### PR TITLE
Added translation greek

### PR DESCRIPTION
**Pull request by Pyrofanis**
## Description

added the greek translation /data/po/el.po
and added an entry for greek in the section of SUPPORTED_LOCALES in minigalaxy/constants.py

## Checklist
 
 - [ ] _CHANGELOG.md_ was updated (**Greek** (thanks to pyrofanis))

### CODE REVIEW

**Comment by Pyrofanis** [comment]
what should i do to resolve those issues.it is my first public pull request so i am a tiny bit lost

File: minigalaxy/constants.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
     ["es", _("Spanish")],
     ["sv", _("Swedish")],
     ["tr", _("Turkish")],
+
```
Comment (sharkwouter) [inline minigalaxy/constants.py:26]: This empty new empty line is causing an issue with the linting test. Could you remove it?

**Review by sharkwouter — CHANGES_REQUESTED** [review]
Thanks for the PR! It looks pretty good to me, it just needs a tiny fix. See my comment.

Would it be possible to add something like ``- Added Greek translation (thanks to Pyrofanis)`` to the ``CHANGELOG.md`` file?

File: minigalaxy/constants.py (hunk 1 above)
Comment (Pyrofanis) [reply minigalaxy/constants.py:26]: yeah thanks i am at work r8 now going to correct it when i get home :)

**Comment by Pyrofanis** [comment]
i am closing it to assign the new corrected pull request sry for any issues

**Pull request closed without merging**